httpserver/https_client.cpp

```diff
@@ -45,26 +45,26 @@ void make_https_request(boost::asio::io_context& ioc,
     static ssl::context ctx{ssl::context::tlsv12_client};
 
     auto session = std::make_shared<HttpsClientSession>(
-        ioc, ctx, std::move(resolve_results), req, std::move(cb),
+        ioc, ctx, std::move(resolve_results), "service_node", req, std::move(cb),
```

Comment:
This should probably be `service-node` or something like that so that it is a valid hostname.  (Previously it looks like it had a space in the hostname it was setting, but _ isn't a valid hostname character either).

Reply:
Didn't mean to change the hostname for service nodes (the issue was with fileserver requests), but sure, it doesn't hurt to use a valid hostname here too. Updated.